corehq/apps/accounting/subscription_changes.py

```diff
@@ -479,9 +478,9 @@ def response_mobile_worker_creation(self):
                 return
             num_allowed = user_rate.monthly_limit
             num_extra = num_users - num_allowed
-
             if num_extra > 0:
-                if self.new_plan_version != SoftwarePlanEdition.COMMUNITY:
+                from corehq.apps.accounting.models import DefaultProductPlan
+                if self.new_plan_version != DefaultProductPlan.get_default_plan_version():
```

Comment:
What's the distinction here?

Reply:
new_plan_version is an object, so the comparison with the string didn't work